projects/plugins/jetpack/extensions/shared/components/stripe-nudge/style.scss

```diff
@@ -16,6 +16,6 @@
 	padding: 6px;
 }
 
-.jetpack-stripe-nudge__banner span.block-editor-warning__action a {
-	color: var( --color-primary );
+.jetpack-stripe-nudge__banner .jetpack-stripe-nudge__link {
+	color: #0675c4 !important;
```

Comment:
Is there no sass variable instead of HEX?

Reply:
Ah yes, I copied this directly from the browser while testing. Thanks for catching this